### DISCUSSION TITLE
Why spotube play indian song rather than the intended ones

### DISCUSSION THREAD

**Discussion by dan1471** [KRTirtho/spotube#1746] [Q&A]
When I play song the cover is what I want but the song is an Indian song what is the problem?

**Comment by AlexDeFoc** [comment]
> When I play song the cover is what I want but the song is an Indian song what is the problem?

It's cuz it tries to source from "best guess", meaning it sources the best thing matching if it's not the good one then the mimicking of the real thing is down and you gotta manually click other sources or i forgot button, and find the right one, most probably youtube.

**Comment by dan1471** [comment]
Thank you for your replay .another issue is that it takes a long period to
play the next song

On 1 Aug 2024 3:26 am, "Alex" ***@***.***> wrote:

When I play song the cover is what I want but the song is an Indian song
what is the problem?

It's cuz it tries to source from "best guess", meaning it sources the best
thing matching if it's not the good one then the mimicking of the real
thing is down and you gotta manually click other sources or i forgot
button, and find the right one, most probably youtube.

—
Reply to this email directly, view it on GitHub
<https://github.com/KRTirtho/spotube/discussions/1746#discussioncomment-10206291>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/BIDNDPGHROC2EKOHTMBTRH3ZPF6EPAVCNFSM6AAAAABLWHC5IWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMRQGYZDSMI>
.
You are receiving this because you authored the thread.Message ID:
***@***.***>
